src/app/theme/themeManager.ts

```diff
@@ -0,0 +1,65 @@
+// Import the EventEmitter class to enable an event-driven approach.
+// We use events to allow different parts of the application to react to theme changes
+// in a decoupled way. This ensures that React components and other services
+// can both respond to theme changes without tight coupling between them.
+import { EventEmitter } from 'events'
+
+import { ThemeType } from '/app/theme/models'
+import { devlog } from '/core/tools/env'
+import { ThemeInput } from '/libs/intents/setFlagshipUI'
+
+const eventEmitter = new EventEmitter()
+
+// The 'theme' variable holds the current theme state. It's kept in this module so
+// that it's accessible both by React components and other services in a centralized manner.
+let theme: ThemeType = ThemeType.Inverted
+
+/**
+ * The getTheme function allows other parts of the application to retrieve
+ * the current theme.
+ */
+export function getTheme(): ThemeType {
+  devlog('🎨 themeManager', `getTheme: ${theme}`)
```

Comment:
I should have used minilog in these new logs so we can hide them if needed, this is a mistake